### PR TITLE
Setup and mobile version skeleton

### PR DESCRIPTION
**Pull request by gemmen29**
In this project, I made a "Portfolio" page doing the following:

1. Add HTML and CSS files.
2. Use linters.
3. Add .gitignore file.
4. Update README.md file template.
5. Use the Mobile-first technique.
6. Add navbar.
7. Add header Section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <nav>
+        <a href="#"><h1>My Logo</h1></a>
+        <a href="#">
+            <img src="images/Union.png" alt="Union Image">
+        </a>
+    </nav>
```
Comment (mikethreels) [inline index.html:19]: - [ ] Good job on creating the Navbar 👍 however according to the requirements The container of this section should be a header tag `<header>`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                <a href="#">
+                    <img src="images/Medium.svg" alt="Medium icon">
+                </a>
+                <a href="#">
+                    <img src="images/Github.svg" alt="Github icon">
+                </a>
+                <a href="#">
+                    <img src="images/Angelist.svg" alt="Angelist icon">
+                </a>
+            </div>
```
Comment (mikethreels) [inline index.html:44]: - [ ] Great job on implementing all the Icons 💪 According to the requirements however all the Icons should be grouped into an unordered list `<ul>` and the individual icons should be wrapped in a list element `<li>`

File: README.md [hunk 3]
```diff
@@ -1,31 +1,31 @@
...
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- Linters
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://gemmen29.github.io/Portfolio/)
```
Comment (mikethreels) [inline README.md:20]: - [ ] [OPTIONAL] Nice job adding the live Demo 👍 Right now however it is just showing the Readme file in a separate tab. if you are interested I suggest you have a look at [this article](https://lirad.medium.com/two-ways-to-create-a-live-demo-for-your-github-project-5436b048a40e) which explains how you can setup an actual live Demo of your webpage

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi George, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (gemmen29) [reply index.html:19]: Done.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <a href="#"><h1>My Logo</h1></a>
+        <a href="#">
+            <img src="images/Union.png" alt="Union Image">
+        </a>
+    </header>
```
Comment (TedLivist) [inline index.html:19]: - [ ] It appears you have misunderstood the previous reviewer. For semantic reasons, your navbar should contain the `nav` tag. Here, you should still maintain the **`nav`** tag as it was but wrap everything in a **`header`**, like in the snippet below 👇 👇.
```
<header>
  <nav>
    ...
  </nav>
</header>
```

File: README.md [hunk 5]
```diff
@@ -1,31 +1,31 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
```
Comment (TedLivist) [inline README.md:3]: - [ ] **[OPTIONAL]** I would suggest giving your project an appropriate name. **'Project Name'** is merely a dummy text and so is too generic to be your project name

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @gemmen29,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (gemmen29) [reply index.html:19]: Done.

**Pull request merged**